server/src/main/java/io/whitefox/api/deltasharing/server/DeltaSharesApiImpl.java

```diff
@@ -49,8 +60,28 @@ public Response getTableChanges(
 
   @Override
   public Response getTableMetadata(
-      String share, String schema, String table, String startingTimestamp) {
-    return Response.ok().build();
+      String share,
+      String schema,
+      String table,
+      String startingTimestamp,
+      String deltaSharingCapabilities) {
```

Comment:
I can't find this header in the https://github.com/agile-lab-dev/whitefox/blob/b3f1875921c24d72c0bbbeb98f3c8fc6c470f6b2/docs/protocol/delta-sharing-protocol.md file, only in the openapi (because you added it)
Why are we requiring this header and then returning it almost "verbatim"?

Reply:
Yes, it's not present in the protocol but only in the implementation. https://github.com/delta-io/delta-sharing/blob/main/server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala#L301

If the request doesn't contain the header, it will be added to the response setting Parquet as response format.

Reply:
You're the man! Good job, I will review the doc and merge!
Great Work